apps/src/maze/MazeVisualizationColumn.jsx

```diff
@@ -0,0 +1,29 @@
+var msg = require('../locale');
+var commonStyles = require('../commonStyles');
+
+var Visualization = require('./Visualization');
+var StepButton = require('./StepButton');
+var SpellingControls = require('./SpellingControls');
+var GameButtons = require('../templates/GameButtons');
+var BelowVisualization = require('../templates/BelowVisualization');
+
+var MazeVisualizationColumn = function (props) {
+  return (
+    <span>
+      <Visualization/>
+      <GameButtons hideRunButton={props.hideRunButton}>
+        <StepButton showStepButton={props.showStepButton}/>
```

Comment:
I don't love this convention of handing a component a 'show me' or 'hide me'  property, instead of just having the parent render it or not.  I assume we do that here because we actually want to hide the button but keep it available for StudioApp to bind against.  If that's the case, can we change the property from `showStepButton` to something simpler/more imperative like `visible`?